upgrade/UPGRADE-v9.0.0-dev.md

```diff
@@ -1574,6 +1574,17 @@ following methods has changed their interface, update your usages accordingly:
     +       DisplayTimeZoneProviderInterface $displayTimeZoneProvider
         )
     ```
+- `ErrorPagesFacade::__construct()`
+    ```diff
+        public function __construct(
+            $errorPagesDir,
+            Domain $domain,
+            DomainRouterFactory $domainRouterFactory,
+            Filesystem $filesystem,
+    -       ?ErrorIdProvider $errorIdProvider = null
+    +       ErrorIdProvider $errorIdProvider
+        )
+    ```
```

Comment:
This diff is not right, you have changed the order of arguments

Reply:
Are you sure? 
Order of arguments was changed in `NotLogFakeHttpExceptionsErrorListener` which is not listed because it's part of another instruction in this file 
```
these classes were removed so you might need to update your application appropriately:
    - `NotLogFakeHttpExceptionsExceptionListener` (you can use `NotLogFakeHttpExceptionsErrorListener`)
```